src/2025h1/min_generic_const_arguments.md

```diff
@@ -0,0 +1,97 @@
+- Generic parameters are allowed to be used in const generic arguments (e.g. `Foo<{ <T as Trait>::ASSOC_CONST }>`).
+- Users can specify `_` as the argument to a const generic, allowing inferring the value just like with types.
+    - implemented under `feature(generic_arg_infer)` and is relatively close to stabilization
+- Associated const items can introduce generic parameters to bring feature parity with type aliases
+    - implemented under `feature(generic_const_items)`, needs a bit of work to finish it. Becomes significantly more important *after* implementing `min_generic_const_args`
+- Introduce associated const equality bounds, e.g. `T: Trait<ASSOC = N>` to bring feature parity with associated types
+    - implemented under `feature(associated_const_equality)`, blocked on allowing generic parameters in const generic arguments
+
+Allowing generic parameters to be used in const generic arguments is the only part of const generics that requires significant amounts of work while also having significant benefit. Everything else is already relatively close to the point of stabilization. I chose to specify this goal to be for implementing `min_generic_const_args` over "stabilize the easy stuff" as I would like to know whether the implementation of `min_generic_const_args` will surface constraints on the other features that may not be possible to easily fix in a backwards compatible manner. Regardless I expect these features will still progress while `min_generic_const_args` is being implemented.
+
+## Design axioms
+
+- Do not block future extensions to const generics
+- It should not feel worse to write type system logic with const generics compared to type generics
+- Avoid post-monomorphization errors
+- The "minimal" subset should not feel arbitrary
+
+## Ownership and team asks
+
+**Owner:** @BoxyUwU, project-const-generics lead, T-types member
+
+This section defines the specific work items that are planned and who is expected to do them. It should also include what will be needed from Rust teams.
+
+* Subgoal:
+    * Describe the work to be done and use `↳` to mark "subitems".
+* Owner(s) or team(s):
+    * List the owner for this item (who will do the work) or ![Help wanted][] if an owner is needed.
+    * If the item is a "team ask" (i.e., approve an RFC), put ![Team][] and the team name(s).
+* Status:
+    * List ![Help wanted][] if there is an owner but they need support, for example funding.
+    * Other needs (e.g., complete, in FCP, etc) are also fine.
+
+| Subgoal                      | Owner(s) or team(s)       | Notes |
+| ---------------------------- | ------------------------- | ----- |
+| Discussion and moral support | ![Team][] [lang] [types]  |       |
+| Implementation and mentoring | @BoxyUwu                  |       |
+| Implementation               | @camelid @compiler-errors |       |
+| Reviewer                     | @compiler-errors          |       |
```

Comment:
@compiler-errors I put you down here since you wound up doing impl work last time but if you'd rather not be put here that's totally okay and I can remove you. And just generally if either of you would like to not be involved this time that's also okay but both are very welcome to keep going :3